third-party/components/ApprovalCommentForm/ApprovalCommentForm.scss

```diff
@@ -9,6 +9,10 @@
             width: 100%;
         }
 
+        .label {
```

Comment:
The form-elements/TextArea in BRUI does not pass the hideLabel prop down to the child textarea which would have hidden this label